frontend/src/stores/authStore.ts

```diff
@@ -3,6 +3,7 @@ import { User } from 'oidc-client-ts'
 import { acceptHMRUpdate, defineStore } from 'pinia'
 import { computed, ref } from 'vue'
 
+// This looks dangerous
 import { AUTH } from '#src/env.js'
```

Comment:
This can be changed to, if it is compiled their should be a .js but we don't use this anywhere else
```suggestion
import { AUTH } from '#src/env'
```